packages/create-webpack-app/templates/init/default/src/index.ts

```diff
@@ -0,0 +1,2 @@
+// delete this file if Language is Javascript
```

Comment:
We should handle this ourselves. If the language is not javascript, it shouldn't be scaffolded at all.

Reply:
I am working on this currently, after looking at the tests I have realised the default files and will try to add all other files conditionally.
So in a local change where I have implemented this conditional entry point generation, and soon add others like this.